source/guide/3.2/guide-composer.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: bt_wiki
+title: Blueprint Composer User Guide
+category: Guides
+publish: true
+abstract:
+pageord: 800
+
+
+---
+{%summary%} {{page.abstract}}{%endsummary%}
+
+# Overview
+
+The Blueprint Composer is an editor for composing the blueprint.yaml dynamically using a modern Drag and Drop Interface.
+This guide will quickly walk you through the use of the Blueprint Composer.
+
+{%tip title=Tip%}
+The [Terminology Reference Page]({{page.terminology_link}}) will help you understand some of the terms stated in this guide.
+For your convenience, links are supplied throughout the guide to point you to the right term definition in the terminology page.
+{%endtip%}
+
+# Blueprint Composer Components Overview
+
+## Login
+
+### Composer Users
+
+Upon first use of the Blueprint Composer the user will be asked to enter a user-name.
+After first login, the user will be automatically route to the Blueprint Composer UI unless a logout was performed.
+Currently each user can only save one blueprint, modifying that blueprint and saving changes will overwrite that single stored blueprint.
+
+## Composer
+
+### Topology
+
+The Topology page contains draggable nodes, each node represent an actual component (software/hardware/network components).
+
+![Blueprint Composer topology](/guide/images/ui/composer/topology.png)
+
+Each of the nodes has an editable name and may contain [properties]({{page.terminology_link}}#properties), [interfaces]({{page.terminology_link}}#interface) and [relationships]({{page.terminology_link}}#relationship-type).
+The Blueprint Composer interface allows selecting only one node at a time, a selected node gets highlighted and its details displayed on a side properties pane.
```

Comment:
`only` sounds bad here. Something more happy like "To see the properties simply click the node and a panel will appear"